web/components/issues/issue-layouts/filters/header/filters/project.tsx

```diff
@@ -23,9 +23,9 @@ export const FilterProjects: React.FC<Props> = observer((props) => {
   const [itemsToRender, setItemsToRender] = useState(5);
   const [previewEnabled, setPreviewEnabled] = useState(true);
   // store
-  const { getProjectById, workspaceProjectIds } = useProject();
+  const { getProjectById, joinedProjectIds } = useProject();
```

Comment:
Can you rename joinedProjects to all projects.

`{ getProjectById, joinedProjects:allProjects} = useProject();`

Reply:
I think `joineProjectIds` makes more sense as `allProjectIds` should refer to all the public projects, regardless of whether the user has joined them or not. But `joinedProjectIds`, as the name suggests, returns only the projects the user is a part of.

Reply:
Sorry, I misread. I thought you were filtering on all projects.